src/main/service/ClientService.java

```diff
@@ -0,0 +1,32 @@
+package main.service;
+
+import main.entity.Ticket;
+import main.entity.User;
+import main.service.abstracts.AbstractUser;
+
+public class ClientService extends AbstractUser {
+
+    public void getTicket(User user) {
```

Comment:
Client extends User, meaning that client is a user with some extra fields/functionality

And getTicket is a regular getter for class